### ISSUE TITLE
Resize column table with merge cells

### ISSUE THREAD

**Issue by s-renier-taonix-fr** [xdan/jodit#711]
**Jodit Version:**  Playground (3.6.18)

**Browser:**  FF
**OS:**  Linux
**Is React App:**  False

**Code**
In Playground
Insert table 2x2
```
<table style="width: 100%;"><tbody>
<tr>
	<td><br></td>
	<td><br></td></tr>
<tr>
	<td><br></td>
	<td><br></td></tr></tbody></table>
```
Merge all cells of first line
```
<table style="width: 100%;"><tbody>
<tr>
	<td colspan="2"><br></td>
	</tr>
<tr>
	<td><br></td>
	<td><br></td></tr></tbody></table>
```
Resize the column by moving the vertical line of the middle

**Expected behavior:**
```
<table style="width: 100%;"><tbody>
<tr>
	<td colspan="2" style="width: 100%;"><br></td>
	</tr>
<tr>
	<td style="width: 25.7614%;"><br></td>
	<td style="width: 74.2386%;"><br></td></tr></tbody></table>
```
**Actual behavior:**
The width of the 1st line is more than 100% !
```
<table style="width: 100%;"><tbody>
<tr>
	<td colspan="2" style="width: 124.112%;"><br></td>
	</tr>
<tr>
	<td style="width: 25.7614%;"><br></td>
	<td style="width: 74.2386%;"><br></td></tr></tbody></table>
```

**Comment by jiangji1** [comment]
When I merged some cells by dragging it to change its width.
It occurs a bug.
in the file, src/modules/table.ts, it should be modified like the following:
in the function of setColumnWidthByDelta 
line: 914
In the following, what it is your codes before I modified them:
```
for (i = 0; i < box.length; i += 1) {
	w = box[i][j].offsetWidth;
	percent = ((w + delta) / table.offsetWidth) * 100;
	Table.__mark(
		box[i][j],
		'width',
		percent.toFixed(consts.ACCURACY) + '%',
		marked
	);
}
```

after I modified, the codes: 
```
for (i = 0; i < box.length; i += 1) {
	let left = box[i][j - 1]
	, cur = box[i][j]
	if (left === cur) {
		// @ts-ignore
		w += box[i][j].offsetWidth
	} else {
		w = box[i][j].offsetWidth;
	}
	// @ts-ignore
	percent = ((w + delta) / table.offsetWidth) * 100;
	Table.__mark(
		cur,
		'width',
		percent.toFixed(consts.ACCURACY) + '%',
		marked
	);
}
```

**Comment by s-renier-taonix-fr** [comment]
@xdan I think it's ok now with this commit : [Fixed Resize column table #712 ](https://github.com/xdan/jodit/commit/8b52ef46a9a068bbc214cca3722d66d5295746ee)
You can close this issue.
Thank you !
